packages/buildcop/README.md

```diff
@@ -0,0 +1,21 @@
+buildcop
```

Comment:
nit, but we should probably update the upstream template to make this title case:

```
# buildcop
```

Reply:
Updated this readme.